vocab/credentials/v2/issuer-dependent.html

```diff
@@ -66,15 +68,58 @@
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, 
+            as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary 
+            items defined by the Verifiable Credentials Working Group through different vocabularies, 
+            such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. 
+            Applications using additional, user-dependent vocabularies of their own are encouraged to 
+            create their own formal vocabularies, and corresponding JSON-LD context definitions, 
+            to ensure the interoperability of the credentials they produce and/or consume.
         </p>
         <p>
-            There may be cases, however, when the creation of such formal vocabularies and context definitions is too much of a burden, such as for closed applications, experiments, or examples. The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed in such a way that issuers may use new, hitherto undefined terms without further ado, and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
+            Not all users desire to define JSON-LD vocabularies or to make use of them with W3C Verifiable Credentials. 
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed 
+            in such a way that issuers may use new, hitherto undefined terms without further ado, 
+            and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
         </p>
         <p>
             This document provides a proper landing place for those URLs, without any additional definitions for the terms themselves. 
         </p>
+        <section id="dynamic-term" style="color: #FF5C5C;"></section>
     </section>
-
+    <script>
+      function htmlEncode(str){
+          return String(str).replace(/[^\w. ]/gi, function(c){
+              return '&#'+c.charCodeAt(0)+';';
+          });
+      }
+      (async ()=>{
+        // handle issuer / hodler defined terms in fragment
+        if (window.location.hash.length){
+          const [_, term] = window.location.hash.split('#')
+          let el = document.getElementById('dynamic-term')
+          el.id = term;
+          const encodedTerm = htmlEncode(term);
```

Comment:
some fear of reflected XSS here... anyone want to correct this?

Reply:
The `htmlEncode` function you provided does the wrong thing wrt. the following test values: `foo-bar-1`, `foo bar`, `f@@bar`, and so on. 

You could try `encodeURIComponent` instead, which still fails for stuff like `f@@bar`, but works in expected ways for the other two test cases, and `<script>` injection.